collectors/metrics/pkg/forwarder/forwarder.go

```diff
@@ -439,11 +458,13 @@ func (w *Worker) forward(ctx context.Context) error {
 
 	req := &http.Request{Method: "POST", URL: w.to}
 	err = w.toClient.RemoteWrite(ctx, req, families, w.interval)
+	w.metrics.forwardRemoteWriteRequests.Inc()
 	if err != nil {
 		statusErr := w.status.UpdateStatus("Degraded", "Failed to send metrics")
 		if statusErr != nil {
 			rlogger.Log(w.logger, rlogger.Warn, "msg", failedStatusReportMsg, "err", statusErr)
 		}
+		w.metrics.forwardRemoteWriteErrors.Inc()
```

Comment:
do you think its worth adding the status code perhaps to this and the other metrics?

Reply:
Will try!